.gitignore

```diff
@@ -23,3 +23,5 @@ testbin/*
 *.swp
 *.swo
 *~
+
+.DS_Store
```

Comment:
Can remove this file too.

Reply:
which file?

Reply:
I mean the `.DS_Store` file

Reply:
It's mac's system file, tiflow also has this in gitignore.
https://github.com/pingcap/tiflow/blob/master/.gitignore#L29

Reply:
Yep, I mean keep `.DS_Store` in `.gitignore` file, and also remote the file `.DS_Store` from repo

Reply:
Gotta. I will delete them in the next PR.